### ISSUE TITLE
Little Tiles blocks occasionally turn invisible [cosmetic]

### ISSUE THREAD

**Issue by tlitookilakin** [CreativeMD/LittleTiles#18]
sometimes when placing or removing pieces from a blockspace, that block will turn invisible. modifying or otherwise updating the block causes it to render again.

**Comment by CreativeMD** [comment]
Should hopefully be fixed in the newest update. I can't grantee if its fixed alongside Optifine. Just reopen the issue if something is still not working. Thanks for reporting this issue.

**Comment by mankra** [comment]
I have a similar problem. I used LittleTiles to make vertical slabs. Some of the placed slabs are invisible after relogging or reloading the chunk. Which ones is not reproducible. I suspected shaders, but the problem remains after disabling them.

![2017-04-13_01 30 34](https://cloud.githubusercontent.com/assets/22047030/24983978/c3b19bba-1fea-11e7-99aa-c7e85070f829.jpg)

Minecraft: 1.10.2
Forge: 12.18.3.2254
LittleTiles: 1.4.3
CreativeCore: 1.6.26
